j2cl-tasks/src/main/java/com/vertispan/j2cl/build/provided/TranslationsFileProcessor.java

```diff
@@ -45,9 +43,9 @@ public TranslationsFileNotDefined(boolean shouldWarn) {
         }
 
         @Override
-        public Optional<File> getTranslationsFile(List<Input> inputs, BuildLog log) {
+        public Optional<File> getTranslationsFile(List<Input> inputs, TaskContext context) {
             if (shouldWarn) {
-                log.warn("translationsFile only works in the ADVANCED optimization level, in other levels the default messages values will be used");
+                context.log().warn("translationsFile only works in the ADVANCED optimization level, in other levels the default messages values will be used");
```

Comment:
Fun fact, TaskContext extends BuildLog for exactly this kind of situation, so you don't need to chain `.log()` in there